### ISSUE TITLE
Losing money on death regardless of pvp?

### ISSUE THREAD

**Issue by Dragoboss** [diecode/KillerMoney#1]
**Server software:**    Spigot 1.10.2-R0.1-SNAPSHOT
**KillerMoney version:** KillerMoney v3.16 for 1.10
**Ticket type:** Feature request
**Ticket description:** Dear diecode. Could you make changes in your sourcecode to allow players to lose money on death (configurable) regardless of whether it was a PVP kill or another cause?
This way I can add a drain in our server's economy without the need of yet another plugin, plus I think it would be a fair feature for this plugin by itself.

**Expected output:**
If a player dies, for example by falling to their death, substract an pre-configured (ranged?) amount of money from their balance. I.E. between 3 & 6.

**Current output:**
No money is substracted from account upon death unless it was a PVP kill.

**Be sure to thank developer for his work:**
Thank you for you work on this plugin. I like it very much and hope it'll stay around for a long time.

Kind regards,
Drago

**Comment by kgahor** [comment]
@Dragoboss Do you want death cause option in the config for player?
E.g.:
- Falling
- PVP kill
- Mob kill
- ?

**Comment by Dragoboss** [comment]
I personally do not have the need for that however I'd think others might like that, indeed.

I personally just need money to get substracted from the player's account whenever they die and if it was indeed a PVP kill that money would transfer to the killer, otherwise it'd disappear from the game.

Thank you for looking into it!

**Comment by Dragoboss** [comment]
Been two weeks since my last reply, was wondering if you have any news on this @diecode 
Sorry if you feel this is spam / unnecessary bump.

Just figured since you pushed an update yesterday but this was not included nor replied on within this time.

**Comment by kgahor** [comment]
This is on my todo list, please be patient, I am working other projects too. @Dragoboss

**Comment by Dragoboss** [comment]
I'm more than happy to be patient I was just hoping for a reply at that point so I knew you indeed put it on your lists ;)

I am aware of the pressure that comes with being a developer. So take your time to program it and thank you for letting me know the status.

**Comment by kgahor** [comment]
You can transfer the money from victim to killer with current "Cash-transfer" option in PVP. The money lose in another cause will be available soon.

**Comment by Dragoboss** [comment]
I have decided to start working on this feature myself.
In fact, I feel like I am close to finishing it except for the fact my events do not fire yet.

https://github.com/Dragoboss/KillerMoney/commit/1b43ef22248dac018c5bb85c0a9f70f489e128d6
My commit for adding this as far as I am right now.

If I can actually figure this out and I am done at some point I will send you a Pull Request, it is up to you whether or not you wish to accept it at that point and close this issue report.
I just decided I'd keep you informed.
